content/study/posts/liberia_custom-mb.json

```
@@ -0,0 +1,60 @@
+{
+  "version": 8,
+  "sources": {
+    "transmission_lines_1": {
+      "data": [
+        "https://energydata.info/dataset/AICD-AFR-62/resource/029caf85-1dca-4a76-b51b-54dbe51b3bf8/download/liberia-electricity-transmission-network.geojson"
+      ],
+      "type": "geojson"
+    },
+    "transmission_lines_2": {
+      "data": [
+        "https://development-data-hub-s3-public.s3.amazonaws.com/ddhfiles/145281/liberiafinal.geojson"
+      ],
```

```suggestion
      "data": "https://development-data-hub-s3-public.s3.amazonaws.com/ddhfiles/145281/liberiafinal.geojson",
```

Same problem as above.